### PR TITLE
Create zh_hk.json

### PR DESCRIPTION
**Pull request by SolidBlock-cn**
add language file

### CODE REVIEW

**Review by frqnny — APPROVED** [review]

File: fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "pack.source.fabricmod": "Fabric模组"
```
Comment (liach) [inline fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/zh_cn.json:2]: Why no space here while space is present between character and words in other languages?

Comment (SolidBlock-cn) [reply fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/zh_cn.json:2]: I'm just following the vanilla convention. In vanilla assets files, `zh-cn.json` usually does not leave spaces between character and words, while `zh-hk.json` and `zh_tw.json` do.

Comment (SolidBlock-cn) [reply fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/zh_cn.json:2]: I'm just following the vanilla convention. In vanilla assets files, `zh-cn.json` usually does not leave spaces between character and words, while `zh-hk.json` and `zh_tw.json` do.

File: fabric-item-groups-v0/src/main/resources/assets/fabric/lang/lzh.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "fabric.gui.creativeTabPage": "卷%2$s之%1$s"
```
Comment (liach) [inline fabric-item-groups-v0/src/main/resources/assets/fabric/lang/lzh.json:2]: > I'm just following the vanilla convention. In vanilla assets files, `zh-cn.json` usually does not leave spaces between character and words, while `zh-hk.json` and `zh_tw.json` do.

This one has inconsistency with the other `lzh.json` then

Comment (SolidBlock-cn) [reply fabric-item-groups-v0/src/main/resources/assets/fabric/lang/lzh.json:2]: That's also the convention of vanilla's `lzh.json`, where spaces are usually not left. And actually whether to leave space does not matter—both behaviours are acceptable.

Comment (liach) [reply fabric-item-groups-v0/src/main/resources/assets/fabric/lang/lzh.json:2]: So you add space for english but not number? sure... cursed enough

**Review by liach — APPROVED** [review]

**Review by SolidBlock-cn — COMMENTED** [review]
-

**Comment by Player3324** [comment]
Looks like this was forgotten, why close?

**Pull request closed without merging**